gap/oper.gi

```diff
@@ -309,6 +309,36 @@ function(D, u, v)
   return D;
 end);
 
+InstallMethod(DigraphContractEdge,
+"for a mutable digraph by out-neighbours and two positive integers",
+[IsMutableDigraph and IsDigraphByOutNeighboursRep, IsPosInt, IsPosInt],
+function(D, src, ran)
+  local V, P;
+  if IsMultiDigraph(D) then
+    ErrorNoReturn("the 1st argument <D> must be a digraph with no multiple ",
+                  "edges,");
+  elif not (IsPosInt(src) and src in DigraphVertices(D)) then
```

Comment:
Given the filter of this method, `src` and `ran` are guaranteed to be positive integers, so I will remove this check. In any case, if `src` ever wasn't a positive integer, then it wouldn't be a member of `DigraphVertices(D)`, and so there still wouldn't be a need to check whether it was a positive integer first.
```suggestion
  elif not src in DigraphVertices(D) then
```